ksql-functional-tests/src/test/resources/query-validation-tests/array.json

```diff
@@ -0,0 +1,54 @@
+{
+  "comments": [
+    "Tests covering the use of the array returning UDFs."
```

Comment:
nit: I'm not sure I see why these two should be grouped but explode is in its own test file. I'd prefer having a test file for each UDTF to keep it simple

Reply:
This is following the pattern of math.json which contains all the maths UDFs. Funnily enough I originally had round.json in it's own separate file but was asked to put it together with the other math ones. The idea here was to follow that pattern with all the array functions.

I don't mind either way, but we should decide what is the best approach.